### ISSUE TITLE
SyntaxError in reports.py, line 131. Linux Mint 18.3, Python 2.7.12 & 3.5.2

### ISSUE THREAD

**Issue by marshwiggle** [projecthamster/hamster#388]
I'have just installed hamster on my  laptop (commit 5fcdfa503c35a10795b2d4a79578222a1b5b130).

When I type `hamster` in the command prompt I've got an error:

```
+ ~ $ hamster
Traceback (most recent call last):
  File "/usr/bin/hamster", line 30, in <module>
    from hamster import client, reports
  File "/usr/lib/python3.5/dist-packages/hamster/reports.py", line 131
    """
      ^
SyntaxError: invalid syntax
```
I can't understand what causes this issue, but when I comment out the following lines in the 'reports.py' it starts without any problem:
```
        event_str = f"""\
                     BEGIN:VEVENT
                     CATEGORIES:{fact.category}
                     DTSTART:{fact.start_time}
                     DTEND:{fact.end_time}
                     SUMMARY:{fact.activity}
                     DESCRIPTION:{fact.description}
                     END:VEVENT
                     """

```

**Comment by ederag** [comment]
Thanks for the report.
Should be fixed with #389.

**Comment by ederag** [comment]
OP system was Linux Mint 18.3, python 3.5.2.
